contrib/seeds/README.md

```diff
@@ -5,8 +5,8 @@ Utility to generate the seeds.txt list that is compiled into the client
 
 The seeds compiled into the release are created from the current protx list, like this:
 
-```bash
-dash-cli protx list valid 1 2018966 > protx_list.json
+```sh
+dash-cli protx list valid 1 _TIP_ > protx_list.json
```

Comment:
The note above is not accurate too actually, it must be a list from a "safe" block, not a "current" one. I'd still prefer to revert these changes and fix it in a separate PR.

Reply:
I doesn't like hard-coded `2018966` because it's easy to copy-paste and use same `2018966` for next time.

Reply:
I agree but this PR is a wrong place to fix it